hack/make-rules/test-cmd-util.sh

```diff
@@ -1050,48 +1050,6 @@ run_kubectl_create_filter_tests() {
   set +o errexit
 }
 
-run_kubectl_apply_deployments_tests() {
```

Comment:
Old servers still exist, and the ability to explicitly send a null field is generally important